tests/apollo/util/eliot_logging.py

```diff
@@ -0,0 +1,40 @@
+from eliot import start_action, start_task, to_file, add_destinations, log_call
+from datetime import datetime
+import os
+import sys
+
+
+def set_file_destination():
+    storage_type = os.environ.get('STORAGE_TYPE')
+    tests_names = [m for m in sys.modules.keys() if "test_" in m]
```

Comment:
Note that `test_names` is a generator (so it's a sort of stateful object). When you invoke len(), it will reach its end, so your else statement might behave strangely. I'd suggest to transform it into a list:
``test_names = list([m for m in sys.modules.keys() if "test_" in m]) > 1
```